wiki/Game_mode/osu!/id.md

```diff
@@ -0,0 +1,285 @@
+
+Dengan kata lain: `Akurasi = Jumlah poin dari hit yang berhasil/(Jumlah maksimal hit * 300 poin)`
+
+| Istilah | Rumus |
+| :-: | :-- |
+| **Total poin hit** | `(jumlah perolehan poin yang bernilai 50 * 50) + (jumlah perolehan poin yang bernilai 100 * 100) + (jumlah perolehan poin yang bernilai 300 * 300)` |
+| **Total hit** | `Miss + jumlah perolehan poin yang bernilai 50 + jumlah perolehan poin yang bernilai 100 + jumlah perolehan poin yang bernilai 300` |
+
+Dengan kata lain, setiap *Miss* akan dihitung sebagai 0 poin masing-masing 50 dihitung sebagai 50 poin, masing-masing 100 sebagai 100 poin dan masing-masing 300 sebagai 300 poin.
+
+setelah itu, bagi total poin yang didapat dari hit dengan total hit yang dikalikan dengan 300.
+
+- Jika seorang pemain mendapatkan 300 poin pada semua objek permainan yang ada: maka hasil perhitungan rumus diatas akan sama dengan 1.00, yang menandakan akurasi yang sempurna (100%).
+- Jika seorang pemain mendapatkan 100 pada semua objek permainan yang ada: Hasil perhitungan rumus di atas akan sama dengan 0.3333 (100/300), yang menandakan akurasi senilai 33.33%.
+
+### Skor
+
+Berikut adalah cara menghitung penilaian skor yang diberikan oleh setiap hit circle dan slider:
+
+`Skor = Nilai Hit + (Nilai Hit * ((Combo multiplier * Difficulty multiplier * Mod multiplier) / 25))`
+
+| Istilah | Penjelasan |
+| :-: | :-- |
+| **Nilai Hit** | Skor hit circle (50, 100 atau 300), slider tick dan bonus skor dari spinner |
+| **Combo multiplier** | (Kombo sebelum hit saat ini - 1) atau 0; manapun yang lebih tinggi |
+| **Difficulty multiplier** | Tingkat pengaturan kesulitan untuk beatmap (penjelasan lengkapnya dapat dilihat pada topik berikutnya) |
+| **Mod multiplier** | Pengganda dari mod yang dipilih |
+
+Di samping itu, setiap awal slider, akhir slider, dan *repeat* pada slider yang berhasil dikenai akan bernilai 30 poin; setiap slider tick yang berhasil dikenai akan bernilai 10 poin; dan setiap putaran pada spinner akan bernilai 100 poin.
+
+Sedangkan pada bagian spinner, bonus tambahan senilai 1.000 poin akan diberikan ketika seorang pemain berhasil menyelesaikan spinner dengan sempurna.
+
+#### Perhitungan Difficulty multiplier
+
+[Circle Size (CS)](/wiki/Beatmap_Editor/Song_Setup), [HP Drain (HP)](/wiki/Beatmap_Editor/Song_Setup) dan [Overall Difficulty (OD)](/wiki/Beatmap_Editor/Song_Setup) masing-masing memiliki pengaruhnya tersendiri pada *difficulty point*.
+
+*Difficulty point* yang terkumpul juga dapat mempengaruhi **Difficulty multiplier** dengan ketentuan sebagai berikut sebagai berikut:
+
+| Difficulty point | Difficulty multiplier |
+| :-: | :-- |
+| 0 - 5 | 2x |
+| 6 - 12 | 3x |
+| 13 - 17 | 4x |
+| 18 - 24 | 5x |
+| 25 - 30 | 6x |
+
+Nilai *difficulty point* tertinggi yang dapat diperoleh melalui editor adalah 27 poin pada pengaturan CS7, OD10, dan HP10. Sebaliknya, nilai *difficulty point* terendah yang dapat diperoleh melalui editor adalah 2 pada pengaturan CS2, OD0, dan HP0.
```

Comment:
```suggestion
Nilai patokan *difficulty point* tertinggi yang dapat diperoleh melalui editor adalah 27 poin pada pengaturan CS7, OD10, dan HP10. Sebaliknya, nilai patokan *difficulty point* terendah yang dapat diperoleh melalui editor adalah 2 pada pengaturan CS2, OD0, dan HP0.
```

Reply:
missing `limited` translation

Reply:
instead adding "patokan" i will change this to "Batas nilai *difficulty point* tertinggi ..."